### PR TITLE
Fix Credo Issues

### PR DESCRIPTION
**Pull request by JettChenT**

### CODE REVIEW

**Comment by milesmcc** [comment]
Great work so far!

File: platform/lib/platform/material.ex [hunk 1]
```diff
@@ -110,14 +110,14 @@ defmodule Platform.Material do
     project_id = Keyword.get(opts, :restrict_to_project_id)
 
     filter_user =
-      if not is_nil(user) do
+      if user do
```
Comment (milesmcc) [inline platform/lib/platform/material.ex:113]: Not quite equivalent. Per our conversation, can we do `is_nil` and then flip the if/else?

File: platform/lib/platform/material.ex [hunk 2]
```diff
@@ -110,14 +110,14 @@ defmodule Platform.Material do
...
         dynamic([m, update: u], u.user_id == ^user.id)
       else
         true
       end
 
     filter_project_id =
-      if not is_nil(project_id) do
+      if project_id do
```
Comment (milesmcc) [inline platform/lib/platform/material.ex:120]: Same here (and for all is_nil changes)

File: platform/lib/platform_web/live/media_live/upload_version_live.ex [hunk 3]
```diff
@@ -251,7 +251,7 @@ defmodule PlatformWeb.MediaLive.UploadVersionLive do
             class="w-full flex justify-center items-center px-6 pt-5 pb-6 border-2 h-40 border-gray-300 border-dashed rounded-md"
             phx-drop-target={@uploads.media_upload.ref}
           >
-            <%= live_file_input(@uploads.media_upload, class: "sr-only") %>
+            <.live_file_input upload={@uploads.bulk_upload} class="sr-only" />
```
Comment (milesmcc) [inline platform/lib/platform_web/live/media_live/upload_version_live.ex:254]: Does this work? Should the variable be `media_upload`?

**Review by milesmcc — CHANGES_REQUESTED** [review]
Nearly there! Thanks.

File: platform/lib/platform/material.ex (hunk 1 above)
Comment (JettChenT) [reply platform/lib/platform/material.ex]: changed to `is_nil` on the latest commit.

**Review by milesmcc — APPROVED** [review]

**Pull request merged**